ext/mro/mro.pm

```diff
@@ -89,7 +89,7 @@ resolution order under multiple inheritance. It was first introduced in
 the language Dylan (see links in the L</"SEE ALSO"> section), and then
 later adopted as the preferred MRO (Method Resolution Order) for the
 new-style classes in Python 2.3. Most recently it has been adopted as the
-"canonical" MRO for Perl 6 classes, and the default MRO for Parrot objects
+"canonical" MRO for Raku classes, and the default MRO for Parrot objects
```

Comment:
Maybe we should get rid of that Parrot reference while we're at it?

Reply:
Why not do parrot as its own item?

Reply:
Agreed.

Reply:
Resolved in recent commit.